documentation/en/join-devnet-cli.md

```diff
@@ -1,12 +1,12 @@
-# Join Lotus Devnet
+# Join DevNet
 
-## Node CLI setup
+## Setup
 
-If you have run lotus before and want to remove all previous data: `rm -rf ~/.lotus ~/.lotusstorage`
+If you have run Lotus before and want to remove all previous data: `rm -rf ~/.lotus ~/.lotusstorage`
 
 ## Genesis & Bootstrap
 
-The current lotus build will automatically join the lotus Devnet using the genesis and bootstrap files in the `build/` directory. No configuration is needed.
+The current Lotus build will automatically join the **Lotus DevNet** using the genesis and bootstrap files in the `build/` directory using a default configuration.
```

Comment:
genesis and bootstrap files in `build/` are the default configuration, so it's a bit like saying

> The current Lotus build will automatically join the **Lotus DevNet** using a default configuration using a default configuration.